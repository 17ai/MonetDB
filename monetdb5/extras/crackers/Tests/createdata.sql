create table T1 (a int not null);
copy 1000 records into T1 from stdin USING DELIMITERS '|','\n';
610
634
337
780
908
556
868
305
252
388
703
692
799
767
783
28
521
221
470
639
58
198
715
940
657
464
176
53
288
278
17
947
454
637
890
953
409
315
329
417
967
376
512
160
884
918
103
749
602
430
406
835
759
126
14
623
342
432
973
729
695
938
274
961
517
385
690
941
328
31
723
522
70
97
618
838
308
946
4
201
915
855
975
151
204
491
768
785
731
185
821
710
663
71
714
131
472
983
143
26
944
47
937
685
287
716
272
678
513
397
411
34
807
779
507
202
506
443
717
770
730
247
631
320
261
589
619
341
863
150
220
317
935
773
80
29
229
144
998
509
484
635
96
880
611
795
615
325
987
442
156
286
383
854
836
979
5
948
994
930
901
617
889
567
79
350
747
157
191
566
372
858
364
806
291
21
267
898
217
478
66
25
888
38
257
574
508
280
360
413
642
292
776
410
575
671
550
161
955
570
894
380
535
203
56
892
870
875
822
816
746
361
262
879
333
412
761
664
419
791
597
455
818
145
436
456
620
837
314
64
89
976
13
270
843
351
630
989
374
786
427
211
285
60
511
625
195
88
968
905
284
819
841
897
789
659
533
355
170
916
276
538
416
183
515
927
687
194
712
921
861
458
912
352
497
932
387
110
259
322
386
398
667
510
860
490
479
852
552
612
81
797
87
48
558
136
8
486
933
465
181
718
133
830
757
495
206
118
487
215
63
381
219
665
923
245
655
41
439
105
474
964
293
993
534
85
696
572
707
265
493
346
394
529
632
640
931
378
864
137
353
246
426
591
421
177
751
957
242
728
943
565
104
271
926
966
404
123
560
2
295
224
483
102
573
804
774
6
166
805
736
942
12
545
239
910
919
340
516
392
142
98
390
607
848
576
519
599
164
199
638
742
963
929
332
561
120
122
857
356
457
254
827
971
57
323
444
180
520
461
266
208
764
661
899
344
672
590
445
580
113
210
600
539
569
115
713
877
986
531
368
109
876
913
939
499
741
653
349
59
886
856
914
294
228
169
282
725
904
859
839
313
532
296
956
238
873
544
134
980
518
656
311
425
138
1000
530
626
735
336
312
251
750
369
62
55
660
290
173
467
505
415
273
651
594
907
438
733
689
781
310
182
726
922
428
101
281
471
895
33
46
834
460
803
722
258
568
45
86
793
847
90
379
582
755
777
67
577
991
264
75
463
564
399
559
1
622
236
958
705
524
92
72
382
753
477
184
810
43
263
178
302
878
869
132
450
447
903
969
996
596
400
608
593
139
528
721
441
82
154
125
232
225
758
503
681
130
752
677
481
906
902
434
833
796
547
553
117
709
683
414
65
829
149
885
289
343
578
525
613
403
900
684
866
542
299
934
437
881
384
301
222
158
965
727
462
9
562
153
845
738
321
646
277
702
279
792
316
106
32
682
52
874
212
595
982
283
335
422
163
605
30
924
650
172
633
250
686
652
586
862
234
936
42
74
255
50
896
959
748
621
587
40
375
813
197
911
318
309
186
826
603
766
551
775
737
339
756
84
788
945
688
275
990
720
466
808
719
35
36
995
691
571
557
853
540
452
585
347
269
962
345
227
407
196
190
541
68
359
429
213
121
338
954
480
887
488
872
334
840
3
536
162
614
537
658
231
389
778
249
584
744
909
10
248
371
698
740
598
865
193
760
473
694
977
514
300
616
303
641
326
449
697
739
148
152
636
7
453
124
307
492
146
814
496
396
187
700
654
489
972
297
701
214
648
583
794
670
920
19
268
790
526
579
704
500
825
240
763
175
18
459
147
119
331
324
606
476
171
882
812
15
984
448
997
771
592
188
420
546
679
446
824
408
423
555
366
77
327
401
218
871
230
985
367
588
243
782
928
165
54
298
111
44
842
49
207
69
51
581
950
817
306
78
527
668
200
179
798
680
226
159
76
256
960
174
504
708
674
363
699
127
253
373
244
669
319
883
260
192
693
988
978
647
604
745
73
543
440
925
20
330
395
554
644
846
468
981
485
24
762
112
917
39
402
893
189
549
475
673
22
754
662
999
711
676
168
451
209
11
100
431
116
743
832
502
974
548
601
784
237
828
95
844
609
433
128
800
424
304
435
233
801
675
851
391
482
732
628
772
107
16
469
155
850
93
563
114
365
498
811
831
769
891
357
627
235
629
949
61
108
494
849
970
393
377
23
765
135
370
99
951
815
992
167
952
362
241
867
666
734
809
820
83
216
645
823
223
91
523
418
141
706
802
405
643
787
94
129
724
37
358
205
624
140
27
348
354
501
649
