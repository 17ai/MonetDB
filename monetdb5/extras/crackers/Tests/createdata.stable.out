stdout of test 'createdata` in directory 'extras/crackers` itself:


# 14:47:15 >  
# 14:47:15 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/export/scratch2/petraki/MonetDB/holindex/program/var/MonetDB" "--set" "mapi_open=true" "--set" "mapi_port=36478" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_extras_crackers" "--set" "mal_listing=0"
# 14:47:15 >  

# MonetDB 5 server v11.10.0
# This is an unreleased version
# Serving database 'mTests_extras_crackers', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.630 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2012 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://budapest.ins.cwi.nl:36478/
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_history.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_compress.sql
# loading sql script: 18_dictionary.sql
# loading sql script: 19_cluster.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 24_zorder.sql
# loading sql script: 25_debug.sql
# loading sql script: 39_analytics.sql
# loading sql script: 40_geom.sql
# loading sql script: 80_udf.sql
# loading sql script: 99_system.sql

# 14:47:16 >  
# 14:47:16 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=budapest" "--port=36478"
# 14:47:16 >  

#create table T1 (a int not null);
#copy 1000 records into T1 from stdin USING DELIMITERS '|','\n';
#610
#634
#337
#780
#908
#556
#868
#305
#252
#388
#703
#692
#799
#767
#783
#28
#521
#221
#470
#639
#58
#198
#715
#940
#657
#464
#176
#53
#288
#278
#17
#947
#454
#637
#890
#953
#409
#315
#329
#417
#967
#376
#512
#160
#884
#918
#103
#749
#602
#430
#406
#835
#759
#126
#14
#623
#342
#432
#973
#729
#695
#938
#274
#961
#517
#385
#690
#941
#328
#31
#723
#522
#70
#97
#618
#838
#308
#946
#4
#201
#915
#855
#975
#151
#204
#491
#768
#785
#731
#185
#821
#710
#663
#71
#714
#131
#472
#983
#143
#26
#944
#47
#937
#685
#287
#716
#272
#678
#513
#397
#411
#34
#807
#779
#507
#202
[ 1000	]

# 14:47:16 >  
# 14:47:16 >  "Done."
# 14:47:16 >  

